lib/px/utils/config_loader.lua

```diff
@@ -0,0 +1,83 @@
+local _M = {}
+
+function _M.get_configuration(config_file)
+    local http = require "resty.http"
+    local config = require(config_file)
+    local px_logger = require("px.utils.pxlogger").load(config_file)
+    px_logger.debug("Fetching configuration")
+    local cjson = require "cjson"
+    local px_server = config.configuration_server
+    local px_port = config.configuration_server_port
+    local path = '/module'
```

Comment:
not expecting it to change, and it's more an internal thing.
but if you feel strongly about it, I don't really mind.